kubernetes/README.md

```diff
@@ -1058,7 +1058,7 @@ We will be using [Jaeger](https://www.jaegertracing.io/), which is an open-sourc
 In Kubernetes mode, we can use `kumactl install [..]` again to install the pre-configured Jaeger components onto the Kubernetes cluster we have deployed:
 
 ```bash
-$ kumactl install tracing | kubectl apply -f -
+$ kumactl install observability --components jaeger | kubectl apply -f -
 ``` 
```

Comment:
I think changing the doc to just install the whole stack and then just say that jaeger was install in this previous step if enough/simpler